### PR TITLE
Disable snake eater

### PR DESCRIPTION
**Pull request by jacksonargo**
* Disable snake eater from the parts page. (We can re-enable it for certain auth tokens later).
* Update tests to make sure disabled parts are actually disabled.

### CODE REVIEW

File: pkg/api/views.go [hunk 1]
```diff
@@ -21,17 +21,29 @@ func (x PartsHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
...
 
-	allParts := x.Parts.List()
-	rows := make([]tableRow, 0, len(allParts))
-	for _, part := range allParts {
+	parts := x.Parts.List()
+	want := len(parts)
+	for i := 0; i < want; i++ {
+		if parts[i].Validate() == nil &&
+			projects.GetName(parts[i].Project).Archived == false &&
+			projects.GetName(parts[i].Project).Released == true {
```
Comment (0az) [inline pkg/api/views.go:37]:
```suggestion
			!projects.GetName(parts[i].Project).Archived &&
			projects.GetName(parts[i].Project).Released {
```
I'm 50% sure this passes `gofmt`.

File: pkg/api/helpers.go [hunk 2]
```diff
@@ -57,6 +57,16 @@ func jsonEncode(dest io.Writer, src interface{}) bool {
...
+func jsonEncodeIndent(dest io.Writer, src interface{}, prefix, indent string) bool {
+	encoder := json.NewEncoder(dest)
+	encoder.SetIndent(prefix, indent)
+	if err := encoder.Encode(src); err != nil {
+		logger.WithError(err).Error("json.Encode() failed")
+		return false
+	}
+	return true
+}
+
```
[the remark this answers is not in the document]
Comment (jacksonargo) [reply pkg/api/helpers.go:69]: Hmm ya makes sense.

File: pkg/api/views.go (hunk 1 above)
Comment (jacksonargo) [reply pkg/api/views.go:37]: Personally, I find that less readable.

Comment (0az) [reply pkg/api/views.go:37]: I guess it (probably?) gets optimized out anyway.

**Pull request merged**